db/schema.rb

```diff
@@ -612,6 +612,7 @@
     t.string "encrypted_password_digest_multi_region"
     t.string "encrypted_recovery_code_digest_multi_region"
     t.datetime "second_mfa_reminder_dismissed_at"
+    t.boolean "piv_cac_recommended_dismissed", default: false
```

Comment:
Even if we don't need it in the short term, I'd wonder if it makes sense to store this as a `datetime` similar to what we do with `second_mfa_reminder_dismissed_at`, which could give us the option to re-prompt after some amount of time.